### PR TITLE
Implement reply count indicator to track web UI

### PR DESCRIPTION
**Pull request by Tak**
Resurrected by request

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -359,6 +362,13 @@ protected void setIsReply(boolean isReply) {
 
     }
 
+    private void setReplyCount(int repliesCount) {
+        // This label only exists in the non-detailed view (to match the web ui)
+        if (replyCountLabel != null) {
+            replyCountLabel.setText((repliesCount > 1 ? replyCountLabel.getContext().getString(R.string.status_count_one_plus) : NUMBER_FORMAT.format(repliesCount)));
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java:368]: Is there really a need to format a 0? Like, how could it be displayed differently?

File: app/src/main/res/layout/item_status.xml [hunk 2]
```diff
@@ -319,6 +319,16 @@
...
 
+    <TextView
+        android:id="@+id/status_replies"
+        android:layout_width="40dp"
+        android:layout_height="30dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@id/status_reply"
+        app:layout_constraintTop_toBottomOf="@id/status_poll_description"
+        android:textAlignment="viewEnd"
+        android:textSize="?attr/status_text_medium" />
```
Comment (connyduck) [inline app/src/main/res/layout/item_status.xml:330]: I think it looks best when the number is vertically centered next to the button, like this 
```suggestion
    <TextView
        android:id="@+id/status_replies"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:textSize="?attr/status_text_medium"
        app:layout_constraintBottom_toBottomOf="@id/status_reply"
        app:layout_constraintStart_toEndOf="@id/status_reply"
        app:layout_constraintTop_toTopOf="@id/status_reply" />
```

**Review by connyduck — CHANGES_REQUESTED** [review]
Thank you 🙏

**Comment by Tak** [comment]
:+1: updated

**Review by connyduck — APPROVED** [review]
will merge for Tusky 19

**Comment by Tak** [comment]
In that case, I'll PR the fix for preview cards in timelines and rebase this one on top

**Comment by connyduck** [comment]
Can you please resolve the conflicts?

**Comment by Tak** [comment]
Reresolved

**Pull request merged**